src/infra/schemata/moderation.ts

```diff
@@ -1,3 +1,9 @@
+export type moderation_state = {
```

Comment:
Any reason for leaving this on in infra/?

Reply:
After moving stuff into the component source files, not really. I've moved this file into `moderation/schemata.ts` since the moderation components do share all that stuff.